image/project/Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN npm install --production --no-package-lock && \
     [ ! -d /project/function/node_modules ] && mkdir /project/function/node_modules
 
 # Copy the dependencies into an alpine image
-FROM node:12-alpine
-COPY --chown=node:node . /project
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
This is a good idea.  I am going to make the same change to the POC Go stack as well.